### PR TITLE
Add description to units

### PR DESCRIPTION
**Pull request by yngveaasheim**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@olaaun or @gjoranv Please review and merge.

### CODE REVIEW

File: container-core/src/main/java/com/yahoo/metrics/Unit.java [hunk 1]
```diff
@@ -5,53 +5,54 @@
...
-    VERSION(BaseUnit.VERSION),
-    WAKEUP(BaseUnit.WAKEUP);
+    BINARY(BaseUnit.BINARY, "Zero or one. Zero typically indicate \"false\" while one indicate \"true\""),
+    BUCKET(BaseUnit.BUCKET, "A chunk of documents managed by a distributor service"),
+    BYTE(BaseUnit.BYTE, "A collection of 8 bits"),
+    BYTE_PER_SECOND(BaseUnit.BYTE, BaseUnit.SECOND, "A unit of storage capable of holding 8 bits"),
+    CONNECTION(BaseUnit.CONNECTION, "A link used for communication between a client and a server"),
+    DOCUMENT(BaseUnit.DOCUMENT, "Vespa document, a collection of fields defined in a schema file"),
+    DOCUMENTID(BaseUnit.DOCUMENTID, "A unique document identifier"),
+    FAILURE(BaseUnit.FAILURE, "Failures, typically for requests or operations, nodes"),
```
Comment (gjoranv) [inline container-core/src/main/java/com/yahoo/metrics/Unit.java:15]: Is the ", nodes" at the end intentional? If so, I feel something is missing to explain.

File: container-core/src/main/java/com/yahoo/metrics/Unit.java [hunk 2]
```diff
@@ -5,53 +5,54 @@
...
+    FILE(BaseUnit.FILE, "Data file stored on the disk on a node"),
+    FRACTION(BaseUnit.FRACTION, "A value in the range [0..1]. Higher values can occur for some metrics, but would indicate the value is outside of the allowed range."),
+    HIT(BaseUnit.HIT, "Document that meets the filtering/restriction criteria specified by a given query"),
+    HIT_PER_QUERY(BaseUnit.HIT, BaseUnit.QUERY, "The aggregate number og hits for each query over a period of time"),
```
Comment (gjoranv) [inline container-core/src/main/java/com/yahoo/metrics/Unit.java:19]: s/og/of/

**Review by gjoranv — DISMISSED** [review]
Please consider comments.

File: container-core/src/main/java/com/yahoo/metrics/Unit.java (hunk 1 above)
Comment (yngveaasheim) [reply container-core/src/main/java/com/yahoo/metrics/Unit.java]: Thanks, updated.

File: container-core/src/main/java/com/yahoo/metrics/Unit.java (hunk 2 above)
Comment (yngveaasheim) [reply container-core/src/main/java/com/yahoo/metrics/Unit.java]: Thanks, updated.

**Review by gjoranv — APPROVED** [review]

File: container-core/src/main/java/com/yahoo/metrics/Unit.java [hunk 3]
```diff
@@ -5,53 +5,54 @@
...
-    SECOND(BaseUnit.SECOND),
-    SESSION(BaseUnit.SESSION),
-    TASK(BaseUnit.TASK),
-    THREAD(BaseUnit.THREAD),
-    VERSION(BaseUnit.VERSION),
-    WAKEUP(BaseUnit.WAKEUP);
+    BINARY(BaseUnit.BINARY, "Zero or one. Zero typically indicate \"false\" while one indicate \"true\""),
+    BUCKET(BaseUnit.BUCKET, "A chunk of documents managed by a distributor service"),
+    BYTE(BaseUnit.BYTE, "A collection of 8 bits"),
+    BYTE_PER_SECOND(BaseUnit.BYTE, BaseUnit.SECOND, "A unit of storage capable of holding 8 bits"),
```
Comment (olaaun) [inline container-core/src/main/java/com/yahoo/metrics/Unit.java:11]: Shouldn't this be "Number of bytes per second"?

**Pull request merged**